spec/models/email_processor_spec.rb

```diff
@@ -94,17 +94,37 @@
 
     context "when the entry is a response to yesterday's email" do
       it "sets the entry date to yesterday's date" do
-        yesterday = Date.new(2014, 1, 2)
-        user = create(:user)
-        email = create(
-          :griddler_email,
-          to: [{ token: user.reply_token }],
-          subject: "Re: #{PromptMailer::Subject.new(user, yesterday)}"
-        )
+        Timecop.freeze(2014, 1, 2) do
+          yesterday = Date.new(2014, 1, 1)
+          user = create(:user)
+          email = create(
+            :griddler_email,
+            to: [{ token: user.reply_token }],
+            subject: "Re: #{PromptMailer::Subject.new(user, yesterday)}"
+          )
 
-        EmailProcessor.new(email).process
+          EmailProcessor.new(email).process
 
-        expect(user.newest_entry.date).to eq(Date.new(2014, 1, 2))
+          expect(user.newest_entry.date).to eq(yesterday)
+        end
+      end
+    end
+
+    context "when the entry is a response to an email from last year" do
+      it "sets the entry date to last year" do
+        Timecop.freeze(2015, 1, 1) do # First day of the new year
```

Comment:
How about a local instead of a comment?

```
first_day_of_new_year = Date.new(...)
Timecop.freeze(first_day_of_new_year)
```

I don't think that will work exactly, but that sort of idea.